src/app/app.component.ts

```diff
@@ -273,6 +276,24 @@ export class AppComponent {
     }
 
     this.initPerimeterX();
+
+    this._fullscreen.addFullscreenListener(() => {
+      if (this.isFullscreen) {
+        this.addSkipToMainContent()
+      } else {
+        this.removeSkipToMainContent()
+      }
+    })
+  }
+
+  private removeSkipToMainContent() {
+    let skipToMainContentButton = document.getElementById("skip-main-content-button")
+    skipToMainContentButton.tabIndex = -1
+  }
+
+  private addSkipToMainContent() {
+    let skipToMainContentButton = document.getElementById("skip-main-content-button")
+    skipToMainContentButton.tabIndex = 1
```

Comment:
I might be forgetting something, but was there a reason we are setting the `tabindex` instead of just showing/hiding the button using `ngIf` like we do with the nav/header?

Reply:
Because the skip-to-main-content is written in html (not Angular component), where we can't use `*ngIf`.

Reply:
Here is the code: https://github.com/ithaka/aiw-ui/pull/1210/files/b40cc3cd53daa508bdd5ce2e5bfbcc1f15fe4501#diff-5c26d2c8f8838f32ace87fdecd6344eaL33-R46

Reply:
I think that is an angular component right? It's just written in html form (not pug). I see a few angular specific directives in that block of code you referenced.

Reply:
Ooooops, let me try to use `*ngIf` then!

Reply:
Done!